src/dreyfus/src/clouseau_rpc.erl

```diff
@@ -316,3 +326,49 @@ rpc(Ref, Msg) ->
 
 clouseau() ->
     list_to_atom(config:get("dreyfus", "name", "clouseau@127.0.0.1")).
+
+-type service() :: sup | main | analyzer | cleanup.
+-type liveness_status() :: alive | timeout.
+
+-spec clouseau_services(MajorVsn) -> [service()] when
+    MajorVsn :: binary() | null.
+clouseau_services(MajorVsn) ->
+    case MajorVsn of
```

Comment:
Should we have this case distinction given that both branches are the same at the moment?  I think it is fine to receive the major version as a parameter, but I am not sure if that knowledge should be utilized now.